### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by cjjavellana**
Removed polling & heartbeat responsibility from ClientStore. Moved it to connclient (short for connected client) package. Added Pprof when running in debug mode.

### CODE REVIEW

File: examples/msgproducer/main.py [hunk 1]
```diff
@@ -1,6 +1,57 @@
 import app_pb2
 from kafka import KafkaProducer
 
+dictionary = dict()
```
Comment (cuongnc01655) [inline examples/msgproducer/main.py:4]: Err @cjjavellana you bring our test here

File: internal/messaging/s2s/registry.go [hunk 2]
```diff
@@ -1,7 +1,8 @@
-package messaging
+package s2s
 
 import (
 	"errors"
+	"github.com/yoo/eye/internal/messaging/ipc"
```
Comment (cuongnc01655) [inline internal/messaging/s2s/registry.go:5]: hey @cjjavellana why we need import project prefix in here ?

File: examples/msgproducer/main.py (hunk 1 above)
Comment (cjjavellana) [reply examples/msgproducer/main.py]: Experiments removed.

File: internal/messaging/s2s/registry.go (hunk 2 above)
Comment (cjjavellana) [reply internal/messaging/s2s/registry.go:5]: Import path includes the project's module name.

**Pull request merged**